com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricClouds.compute

```diff
@@ -188,10 +194,18 @@ CloudRay BuildRay(uint2 halfResCoord)
     ray.direction = normalize(pos);
 
     // Compute the max cloud ray length
+#ifdef LOCAL_VOLUMETRIC_CLOUDS
     ray.maxRayLength = lerp(_ProjectionParams.z, length(pos), isOccluded);
+#else
+    ray.maxRayLength = 200000.0f;
```

Comment:
it looks like a bit an arbitrary large number, can we at least setup it under a nameful define? / also add a comment about why this value, thanks

Reply:
Will do

Reply:
@sebastienlagarde done